nova_cli/src/main.rs

```diff
@@ -71,10 +71,19 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             let realm = agent.current_realm_id();
 
             let script = parse_script(&allocator, file.into(), realm, None).unwrap();
-            let result = script_evaluation(&mut agent, script).unwrap();
-
-            if verbose {
-                println!("{:?}", result);
+            match script_evaluation(&mut agent, script) {
+                Ok(result) => {
+                    if verbose {
+                        println!("{:?}", result);
+                    }
+                }
+                Err(error) => {
+                    eprintln!(
```

Comment:
**thought:** I guess eventually this should go into some host hook, maybe?

Reply:
Huh, okay. I hadn't actually realised there's a split between how V8 does tasks and how the spec defines them.